src/main/java/net/sf/jabref/gui/openoffice/OOBibBase.java

```diff
@@ -1366,13 +1366,13 @@ public BibDatabase generateDatabase(List<BibDatabase> databases)
                     BibEntry clonedEntry = (BibEntry) entry.get().clone();
                     clonedEntry.setId(IdGenerator.next());
                     // Insert a copy of the entry
-                    resultDatabase.insertEntryWithDuplicationCheck(clonedEntry);
+                    resultDatabase.insertEntry(clonedEntry);
```

Comment:
Just wondering: Is it save to remove the duplication check here? The documentation of `insertEntryWithDuplicationCheck` says `use insertEntry and DuplicationChecker separately`.

The comment also applies to the replacement below.